recipe/meta.yaml

```diff
@@ -7,14 +9,14 @@ package:
 source:
   fn: django-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/D/Django/Django-{{ version }}.tar.gz
-  sha256: b6f3b864944276b4fd1d099952112696558f78b77b39188ac92b6c5e80152c30
+  sha256: {{ sha256 }}
   patches:
     # Hard-code gdal and geos' paths to the corresponding conda packages.
     - libgdal.patch
     - libgeos.patch
 
 build:
-  number: 0
+  number: {{ build }}
```

Comment:
I've seen jinja templates used for a variety of fields that change between updates. Having the `build` as a template, I believe, makes it more prominent and less likely to be overlooked during subsequent updates.